lua/screen.lua

```diff
@@ -174,11 +174,11 @@ Screen.font_face = function(index) s_font_face(index) end
 -- @param size in pixel height
 Screen.font_size = function(size) s_font_size(size) end
 
--- draw single pixel (requires integer x/y, line_width == 1, stroke afterwards)
+--- draw single pixel (requires integer x/y, fill afterwards)
 -- @param x position
 -- @param y position
-Screen.pixel = function(x,y)
-  s_rect(x-0.25,y-0.25,0.5,0.5)
+Screen.pixel = function(x, y)
+  s_rect(x, y, 1, 1)
```

Comment:
haven't tried this but if x,y really need to be int maybe it would be appropriate to "cast" them"?
`s_rect(int(x), int(y), 1, 1)`

Reply:
Well actually they can be floats you just won't get a sharp pixel (might be useful for animation though). I can remove that comment if it's confusing?